### ISSUE TITLE
Remake Landing Page.

### ISSUE THREAD

**Issue by ADRE9** [ADRE9/bunk-manager-mern#91]
**Deliverables:**
- [x] Remake The landing page to look like [this](https://www.figma.com/file/dQbGpSkclUec07vztFkUEV/Material-Design-UI-kit-Components-library-Community?node-id=120%3A1658)
- [x] Change color palettes.

**Screenshots**
![Screenshot 2021-03-30 225634](https://user-images.githubusercontent.com/49809895/113030450-3afc9900-91ab-11eb-8c53-52ab98610345.jpg)

**Comment by rithgan** [comment]
Can I work on this?

**Comment by ADRE9** [comment]
Yes sure go ahead!. Make sure you use material UI and use the theme.js file to change colors. If any doubt persists , ask the mentors in our bunk manager server.

**Comment by rithgan** [comment]
Hey @ADRE9 , I want to make the pull request but I am having merge conflicts. I want to ask if there is any easy solution for that?

**Comment by VishalPatil18** [comment]
Can I work on this @ADRE9 under GSSoC21?
If yes kindly assign it to me. Thanks!

**Comment by ADRE9** [comment]
@SpecTEviL just check out the Readme file. I guess the landing page has been done. So you can go for any other page you want.
